tools/pdf/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "pdf",
+  "version": "1.0.0",
+  "description": "Generate a PDF copy of the IceRPC docs",
+  "scripts": {
+    "generate-pdf": "ts-node -P tsconfig.pdf.json index.ts",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
This should probably use the same license as the rest of the repo.

EDIT: I would probably just set it private and remove all these fields like the top level one.